lombok.config

```diff
@@ -0,0 +1,4 @@
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```

Comment:
Have you added Lombok annotations and generated code?  We have said we will avoid Lombok code generation where possible in prod code esp things like @Data.  @Getter is ok but we test them using a pojo test class rather than just remove them from scanning.  So I would suggest writing tests where possible and avoid adding this,

Reply:
Yes we are using `@Builder`. Since the mutation score passes locally but not on the pipeline we thought that maybe lombok.config not being enabled was causing it to fail